rmw_cyclonedds_cpp/src/rmw_node.cpp

```diff
@@ -2040,7 +2044,7 @@ extern "C" rmw_ret_t rmw_publish_serialized_message(
   }
 #endif
 
-  const bool ok = (dds_writecdr(pub->enth, d) >= 0);
+  const bool ok = (dds_forwardcdr(pub->enth, d) >= 0);
```

Comment:
Just verifying this: we are switching from `writecdr` to `forwardcdr` because we are setting the timestamp above?  Is there any potential fallout from this change?

Reply:
Apparently not, the same thing was done in #454.